ucm2/alcatel-idol347/HiFi.conf

```diff
@@ -0,0 +1,57 @@
+# Use case configuration for Alcatel OneTouch Idol 3 (4.7").
+# The two speakers/earpieces are connected to external codec (TFA9897)
+# Headphones support requires writing a driver for AK4375
+
+Define {
+	WcdPlaybackPCM "hw:${CardId},1"
+	WcdCapturePCM "hw:${CardId},0"
+}
+<platforms/msm8916/qdsp6-components.conf>
+
+SectionDevice."Speaker" {
+	Comment "Speaker playback"
+
+	EnableSequence [
+		cset "name='Speaker Top Amp Input' Left"
+		cset "name='Speaker Bottom Amp Input' Right"
+		cset "name='Speaker Top Amp Switch' 1"
+		cset "name='Speaker Bottom Amp Switch' 1"
+		cset "name='Speaker Top Mode' Speaker"
+		cset "name='Speaker Bottom Mode' Speaker"
+	]
+
+	DisableSequence [
+		cset "name='Speaker Top Amp Switch' 0"
+		cset "name='Speaker Bottom Amp Switch' 0"
+	]
+
+	Value {
+		PlaybackChannels 2
+		PlaybackPriority 300
+		PlaybackPCM "hw:${CardId},1"
+	}
+}
+
+SectionDevice."Earpiece" {
+	Comment "Earpiece playback"
+
+	EnableSequence [
+		cset "name='Speaker Top Amp Input' Left"
+		cset "name='Speaker Top Amp Switch' 1"
```

Comment:
I think we also should add a `cset "name='Speaker Top Amp Switch' 0"` here just to make sure only one of them is on.